sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md

```diff
@@ -0,0 +1,213 @@
+# Azure Cognitive Language Services Question Answering client library for .NET
+
+Question Answering is a cloud-based API service that lets you create a conversational question-and-answer layer over your existing data. Use it to build a knowledge base by extracting questions and answers from your semi-structured content, including FAQ, manuals, and documents. Answer users’ questions with the best answers from the QnAs in your knowledge base—automatically. Your knowledge base gets smarter, too, as it continually learns from user behavior.
+
+[Source code][questionanswering_client_src] | [Package (NuGet)][questionanswering_nuget_package] | [API reference documentation][questionanswering_refdocs] | [Product documentation][questionanswering_docs] | [Samples][questionanswering_samples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Key Vault secrets client library for .NET with [NuGet][nuget]:
+
+```powershell
+dotnet add package Azure.AI.Language.QuestionAnswering --prerelease
+```
+
+### Prerequisites
```

Comment:
I would move the "Perquisites" section ahead of the "Install the package" section.

Reply:
This is our standard layout in the .NET repo. The prereqs is more about what you need for the subsequent examples, which follow.